rfq/order.go

```diff
@@ -736,8 +790,13 @@ type OrderHandler struct {
 // NewOrderHandler creates a new struct instance.
 func NewOrderHandler(cfg OrderHandlerCfg) (*OrderHandler, error) {
 	return &OrderHandler{
-		cfg:      cfg,
-		policies: lnutils.SyncMap[SerialisedScid, Policy]{},
+		cfg:         cfg,
+		policyStore: cfg.PolicyStore,
+		policies:    lnutils.SyncMap[SerialisedScid, Policy]{},
+		peerAcceptedBuyQuotes: lnutils.SyncMap[
+			SerialisedScid, rfqmsg.BuyAccept]{},
+		peerAcceptedSellQuotes: lnutils.SyncMap[
+			SerialisedScid, rfqmsg.SellAccept]{},
```

Comment:
why are we not initializing `localAcceptedBuyQuotes` / `localAcceptedSellQuotes` ?

Reply:
actually I think the `lnutils.SyncMap` type doesn't need to be initialized, in that case should remove above lines

Reply:
Thanks, this was moved from the manager but indeed we don't need it, so good to remove.